zookeeper-docs/src/main/resources/markdown/zookeeperAdmin.md

```diff
@@ -1740,6 +1740,16 @@ and [SASL authentication for ZooKeeper](https://cwiki.apache.org/confluence/disp
     **New in 3.5.5:**
     TBD
 
+* *sslQuorumReloadCertFiles* :
+    (No Java system property)
+    **New in  3.5.5, 3.6.0:**
+    Allows Quorum SSL keyStore and trustStore reloading when the certificates on the filesystem change without having to restart the ZK process. Default: false
+
+* *client.certReload* :
+    (Java system property: **zookeeper.client.certReload**)
+    **New in 3.5.9:**
```

Comment:
sorry  @mathew-manu , I asked you to document this as "new in 3.5.9", it should be 3.9.0. Can you change it? I can merge after that.

Reply:
Updated the version to 3.9.0